zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -51,10 +51,10 @@ import {
   closeSearchFormOrIgnore,
 } from './pageWrapperScripts';
 
-import { getQueryParams, SearchType } from './search_results/searchResultsScripts';
+import { getQueryParams, SearchType } from '../views/search_results/searchResultsScripts';
 
 import CustomButton from '../components/button/Button.js';
-import LoadingPage from './loading/LoadingPage';
+import LoadingPage from '../views/loading/LoadingPage';
```

Comment:
revert change. PageWrapper is already inside `views` folder. This was introduced by Berra but anyone that is available should please fix it